lib/gather/analyze-ember-object.js

```diff
@@ -2,14 +2,27 @@ module.exports = function analyzeEmberObject(possibleEmberObject) {
   if (typeof possibleEmberObject !== 'object' || possibleEmberObject === null) {
     return undefined;
   }
-  if (possibleEmberObject.default && typeof possibleEmberObject.default.proto !== 'function') {
+  if (
+    possibleEmberObject.default &&
+    (typeof possibleEmberObject.default.proto !== 'function' ||
+      typeof possibleEmberObject.default.create !== 'function')
+  ) {
     return undefined;
   }
-  let proto = possibleEmberObject.default.proto();
+
+  let obj = possibleEmberObject.default.proto
+    ? possibleEmberObject.default.proto()
+    : possibleEmberObject.default.create();
```

Comment:
I don't understand this change, why would we `.create()`?

Reply:
When investing this bug originally I was able to enumerate (or at least I thought I was) more types of objects when using create if proto was not available. If this is not needed I’m still left trying to answer the original question - how do you tell class based helpers from the old(?) style?  Is isHelperFactory a reliable way?

Reply:
> Is isHelperFactory a reliable way?

Yes, that is what I think we should use. I don't think we want to `.create()` things (it is pretty likely to actually cause errors if we `.create()` a thing without the right arguments and stuff).

Reply:
great, i will revert a majority of this and push